js/app/Visualization/UI/UIManager.js

```diff
@@ -186,6 +186,11 @@ define([
       self.loadingNode = $('<div class="loading"><img style="width: 20px;" src="' + Paths.LoaderIcon + '"></div>');
       self.visualization.animations.map.controls[google.maps.ControlPosition.LEFT_TOP].push(self.loadingNode[0]);
 
+      /* Add a class to the zoom buttons so we can modify them with our css */
+
+      $(".gm-bundled-control .gmnoprint:has(div[title='Zoom in'])").addClass('zoomButtons')
```

Comment:
@enriquetuya here is where we add the class to the map zoom buttons.  It is possible that this happens to soon before the buttons have been initialized.